projects/batfish/src/main/java/org/batfish/smt/Encoder.java

```diff
@@ -802,12 +803,12 @@ private BoolExpr environmentBlockingClause(Model m) {
     VerificationStats stats =
         new VerificationStats(numNodes, numEdges, numVariables, numConstraints, time);
 
-    if (ENABLE_DEBUGGING) {
-      System.out.println("Constraints: " + stats.getNumConstraints());
-      System.out.println("Variables: " + stats.getNumVariables());
-      System.out.println("Z3 Time: " + stats.getTime());
-    }
-    // System.out.println("Stats:\n" + _solver.getStatistics());
+    //if (ENABLE_DEBUGGING) {
+    System.out.println("Constraints: " + stats.getNumConstraints());
+    System.out.println("Variables: " + stats.getNumVariables());
+    System.out.println("Z3 Time: " + stats.getTime());
+    System.out.println("Stats: \n" + _solver.getStatistics());
+    //}
```

Comment:
Will this lead to printing things on the console unconditionally?

Reply:
Yes it will!